src/components/Gradebook/index.jsx

```diff
@@ -448,38 +447,38 @@ export default class Gradebook extends React.Component {
                       </div>
                       <StatusAlert
                         alertType="danger"
                         dialog="Error retrieving grade override history."
                         open={this.props.errorFetchingGradeOverrideHistory}
                         dismissible={false}
                       />
-                      { !this.props.errorFetchingGradeOverrideHistory && (
-                      <Table
-                        columns={GRADE_OVERRIDE_HISTORY_COLUMNS}
-                        data={[...this.props.gradeOverrides, {
-                          date: this.state.todaysDate,
-                          grader: this.state.updateUserName,
-                          reason: (<input
-                            type="text"
-                            name="reasonForChange"
-                            value={this.state.reasonForChange}
-                            onChange={value => this.onChange(value)}
-                            ref={(input) => { this.overrideReasonInput = input; }}
-                          />),
-                          adjustedGrade: (
-                            <span>
-                              <input
-                                type="text"
-                                name="adjustedGradeValue"
-                                value={this.state.adjustedGradeValue}
-                                onChange={value => this.onChange(value)}
-                              /> {this.state.adjustedGradePossible}
-                            </span>),
-                        }]}
-                      />)}
+                      {!this.props.errorFetchingGradeOverrideHistory && (
+                        <Table
+                          columns={GRADE_OVERRIDE_HISTORY_COLUMNS}
+                          data={[...this.props.gradeOverrides, {
+                            date: this.state.todaysDate,
+                            grader: this.state.updateUserName,
+                            reason: (<input
+                              type="text"
+                              name="reasonForChange"
+                              value={this.state.reasonForChange}
+                              onChange={value => this.onChange(value)}
+                              ref={(input) => { this.overrideReasonInput = input; }}
+                            />),
+                            adjustedGrade: (
+                              <span>
+                                <input
+                                  type="text"
+                                  name="adjustedGradeValue"
+                                  value={this.state.adjustedGradeValue}
+                                  onChange={value => this.onChange(value)}
+                                /> {this.state.adjustedGradePossible}
+                              </span>),
+                          }]}
+                        />)}
```

Comment:
This is just whitespace format fixed. The real code change is only on line 557 below